logoper.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Logical operators</title>
+</head>
+
+<body>
+    <script>
+        let age = prompt("Какой у вас возраст?", 25);
+        if (age >= 14 && age <= 90) {
+            alert("Включительно");
+        }
+    </script>
+
+    <script>
+        let age = prompt("Какой у вас возраст?", 25);
+        if (!(age >= 14 && age <= 90)) {
+            alert("НЕ Включительно");
+        }
+    </script>
+
+    <script>
+        let login = prompt("Кто там?");
+        if (login === "Админ") {
+            let password = prompt("Пароль?");
+            if (password === "Я главный") {
+                alert("Здравствуйте")
+            } else if (password === " " || password === null) {
```

Comment:
remove this  === ' '
if i write 3 spaces?

rewrite condition without ' ' and null etc.

Reply:
fixed